packages/core/helper-plugin/lib/src/content-manager/utils/formatContentTypeData.stories.mdx

```diff
@@ -0,0 +1,32 @@
+<!--- formatContentTypeData.stories.mdx --->
+
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="utils/formatContentTypeData" />
+
+# formatContentTypeData
+
+This util is used to format the data received by the backend so it can be used by the admin.
+It:
+- hads the key `__temp_key__` to each component (easier for reordering repeatable components)
```

Comment:
```suggestion
- passes the key `__temp_key__` to each component (easier for reordering repeatable components)
```

Reply:
Why do you suggest passes? It adds a key to the component. (I see that I miswrote `adds` :p)

Reply:
(I replaced it with `sets`)